install/Knative-with-IKS.md

```diff
@@ -68,12 +68,16 @@ components, the recommended configuration for a cluster is:
     ```bash
     ibmcloud cs region-set $CLUSTER_REGION
     ```
+1.  Select a Kubernetes version:
+    ```bash
+    ibmcloud cs kube-versions
+    ```
 1.  Create a Kubernetes cluster on IKS with the required specifications:
 
     ```bash
     ibmcloud cs cluster-create --name=$CLUSTER_NAME \
       --zone=$CLUSTER_ZONE \
-      --kube-version=1.10.3 \
+      --kube-version=$CLUSTER_K8S_VERSION \
```

Comment:
Do you need an additional step to set this env var?

Reply:
Thanks for the review! 
I though it would be clear from the description in the previous step (similar as it's done later for PRIVATE_VLAN_ID and PUBLIC_VLAN_ID, but I would be happy to add a not in the previous step to make this more clear.